src/PayID/pay-id-client.ts

```diff
@@ -22,7 +24,45 @@ export default class PayIDClient {
    * @param payID The payID to resolve for an address.
    * @returns An XRP address representing the given PayID if one exists, otherwise undefined.
    */
-  public async xrpAddressForPayID(_payID: string): Promise<string | undefined> {
-    throw PayIDError.unimplemented
+  public async xrpAddressForPayID(payID: string): Promise<string | undefined> {
+    const paymentPointer = PayIDUtils.parsePaymentPointer(payID)
+    if (!paymentPointer) {
+      throw PayIDError.invalidPaymentPointer
+    }
+
+    const url = `https://${paymentPointer.host}${paymentPointer.path}`
```

Comment:
Does `path` here include the leading `/`? Or does `host` have a trailing `/`?

Reply:
Path has a leading /. 

This is following the conventions set in Quilt (which is the implementation I'm mirroring) that `path` always starts with `/` .

Invariant explicitly defined in Quilt Code: https://github.com/hyperledger/quilt/blob/master/spsp-parent/spsp-core/src/main/java/org/interledger/spsp/PaymentPointer.java#L90

Tests showing this is the case: https://github.com/hyperledger/quilt/blob/master/spsp-parent/spsp-core/src/test/java/org/interledger/spsp/PaymentPointerTest.java